version.bzl

```diff
@@ -10,13 +10,13 @@ the app (that are potentially not broadly released yet).
 """
 
 MAJOR_VERSION = 0
-MINOR_VERSION = 11
+MINOR_VERSION = 13
```

Comment:
We skipped 12?

Reply:
I for sure forgot to bump it in the Bazel file during the last release; I'm not sure if I forgot to update it locally during the build (though it doesn't really matter anymore). More reasons to move to automation I suppose.